utils/prune_binaries.py

```diff
@@ -16,47 +16,101 @@
     'build/linux/debian_bullseye_amd64-sysroot/',
+    'build/linux/debian_bullseye_arm64-sysroot/',
+    'build/linux/debian_bullseye_armhf-sysroot/',
     'build/linux/debian_bullseye_i386-sysroot/',
-    'third_party/js_code_coverage/',
-    'third_party/openscreen/src/buildtools/linux64/format/',
+    'build/linux/debian_bullseye_mips64el-sysroot/',
+    'build/linux/debian_bullseye_mipsel-sysroot/',
     # other
     'third_party/depot_tools/external_bin/',
+    # Match removals for the tarball:
+    # https://source.chromium.org/chromium/chromium/tools/build/+/main:recipes/recipe_modules/chromium/resources/export_tarball.py
+    'base/tracing/test/data/',
+    'chrome/test/data/',
+    'components/test/data/',
+    # prune the base path instead of individual ones since clones skip them
+    'content/test/data/',
+    'courgette/testdata/',
+    'extensions/test/data/',
+    'media/test/data/',
+    'native_client/src/trusted/service_runtime/testdata/',
+    'testing/libfuzzer/fuzzers/wasm_corpus/',
+    'third_party/blink/perf_tests/',
+    'third_party/blink/tools/',
+    'third_party/blink/web_tests/',
+    'third_party/breakpad/breakpad/src/processor/testdata/',
+    'third_party/catapult/tracing/test_data/',
+    'third_party/dawn/test/',
+    'third_party/expat/src/testdata/',
+    'third_party/harfbuzz-ng/src/test/',
+    'third_party/hunspell/tests/',
+    'third_party/hunspell_dictionaries/',
+    'third_party/jdk/current/',
+    'third_party/jdk/extras/',
+    'third_party/liblouis/src/tests/braille-specs/',
+    'third_party/llvm/llvm/test/',
+    'third_party/ots/src/tests/fonts/',
+    'third_party/rust-src/src/gcc/gcc/testsuite/',
+    'third_party/rust-src/src/llvm-project/clang/test/',
+    'third_party/rust-src/src/llvm-project/llvm/test/',
+    'third_party/screen-ai/linux/resources/',
+    'third_party/sqlite/src/test/',
+    'third_party/swiftshader/tests/regres/',
+    'third_party/test_fonts/test_fonts/',
+    'third_party/xdg-utils/tests/',
+    'tools/perf/testdata/',
+    'v8/test/',
     # lite tarball paths:
     # https://source.chromium.org/chromium/chromium/tools/build/+/main:recipes/recipes/publish_tarball.py
     'android_webview/',
     'buildtools/reclient/',
+    'chrome/android/',
     'chromecast/',
+    'ios/',
     'native_client/',
     'native_client_sdk/',
     'third_party/android_platform/',
+    'third_party/angle/third_party/VK-GL-CTS/',
     'third_party/apache-linux/',
     'third_party/blink/manual_tests/',
     'third_party/blink/perf_tests/',
     'third_party/catapult/third_party/vinn/third_party/v8/',
+    'third_party/closure_compiler/',
     'third_party/instrumented_libs/',
-    'third_party/llvm/',
-    'third_party/llvm-build/',
     'third_party/llvm-build-tools/',
+    'third_party/llvm-build/',
+    'third_party/llvm/',
     'third_party/node/linux/',
     'third_party/rust-src/',
     'third_party/rust-toolchain/',
     'third_party/webgl/',
 )
```

Comment:
I'd suggest keeping the lite/tarball sections of this list as near-copies of the upstream lists, just to make cross-checking and future merges easier to deal with. That's why I moved the `VK-GL-CTL` entry, etc.

Reply:
@iskunk was your concern addressed?

Reply:
> @iskunk was your concern addressed?

Yes, @Ahrotahn took the maintainability aspect into account. (I'm planning on at least one more change to the upstream lists, after all.)

I would have resolved this conversation, but I don't have the button for it...

Reply:
Sorry, it seems that was available for me for some reason...